src/EA.Weee.Web/Views/Shared/_Layout.GovUK.cshtml

```diff
@@ -50,7 +50,7 @@
     <header class="govuk-header @Html.Raw(ViewBag.HeaderClass ?? string.Empty)" role="banner" data-module="govuk-header">
         <div class="govuk-header__container govuk-width-container">
             <div class="header-global">
-                <div class="govuk-header__logo">
+                <div class="govuk-header__logo" style="width: 200px;">
```

Comment:
have a look at the _index.css, there is a padding set in the class govuk-header__logo

also that logo is used in Maintenance.cshtml file. Might need a change there too.